src/main/java/org/jenkinsci/plugins/badge/BadgeActionFactory.java

```diff
@@ -16,22 +16,42 @@
 @Extension
 public class BadgeActionFactory extends TransientActionFactory<Job> {
 
+    /**
+     * TO DO
+     */
     private final ImageResolver iconResolver;
 
+    /**
+     * TO DO
+     * @throws IOException 
+     */
     public BadgeActionFactory() throws IOException {
         iconResolver = new ImageResolver();
     }
 
+    /**
+     * TO DO
+     * @return
+     */
     @Override
     public Class<Job> type() {
         return Job.class;
     }
 
+    /**
```

Comment:
[Checkstyle] WARNING: First sentence of Javadoc is incomplete (period is missing) or not present.